modules/API/external/external_api.js

```diff
@@ -207,11 +206,36 @@ export default class JitsiMeetExternalAPI extends EventEmitter {
      * @private
      */
     _createIFrame(height, width) {
+        // Compute valid values for height and width. If a number is specified
+        // it's treated as pixel units and our minimum constraints are applied.
+        // If the value is expressed in em, pt or percentage, it's used as is.
+        // Also protect ourselves from undefined, because
+        // Math.max(undefined, 100) === NaN, obviously.
+        //
+        // This regex parses values of the form 100em, 100pt or 100%. Values
+        // like 100 or 100px are handled outside of the regex, and invalid
+        // values will be ignored and the minimum will be used.
```

Comment:
It will be good if we move this to the jsdoc or at least to have something in the jsdoc that describes the supported values. And also maybe we should add this in the api.md. WDYT?